index.html

```diff
@@ -1,214 +1,212 @@
-have to worry about losing your current place.</p>
-
-<h1 id="limitations">Limitations</h1>
-
-<ul>
-  <li>There is only a two-level hierarchy — tabs cannot be the children
-of other tabs in the tree.</li>
-  <li>You cannot open and close individual tabs — you have to open and close
-the window as a whole.</li>
-  <li>Windows will not necessarily open at the size of the last-closed window,
-like they do normally in Chrome.</li>
-</ul>
-
-<p>See the <a href="https://github.com/cxw42/TabFern/issues">issue tracker</a> on GitHub for
-more about current limitations, and where TabFern is going in the future!</p>
-
-<h1 id="related-information">Related information</h1>
-
-<ul>
-  <li><a href="/TabFern/privacy.html">Privacy policy</a></li>
-</ul>
-
-<h1 id="thanks">Thanks</h1>
-
-<ul>
-  <li>Inspired by (but not in any way related to or affiliated with)
-<a href="https://chrome.google.com/webstore/detail/tabs-outliner/eggkanocgddhmamlbiijnphhppkpkmkl">Tabs Outliner</a>
-by Vladyslav Volovyk.</li>
-  <li><a href="https://extensionizr.com">Extensionizr</a></li>
-  <li><a href="https://www.jstree.com/">jstree</a></li>
-  <li><a href="https://github.com/alexandernst/jstree-actions">jstree-actions</a></li>
-  <li><a href="https://jquery.com/">jquery</a></li>
-  <li><a href="http://www.chradams.co.uk/fern/maker.html">Barnsley fern generator</a></li>
-  <li><a href="http://www.famfamfam.com/lab/icons/silk/">famfamfam Silk icons</a></li>
-  <li><a href="http://fontawesome.io/">Font Awesome</a></li>
-  <li>Hosted on GitHub Pages — Theme modified from
-<a href="https://github.com/pages-themes/leap-day">Leap Day</a> by
-<a href="https://twitter.com/michigangraham">mattgraham</a></li>
-</ul>
-
-<p><img src="/TabFern/assets/fern128.png" alt="logo" /></p>
-
-<!-- vi: set ts=2 sts=2 sw=2 et ai ft=markdown: -->
-
-
-      </section>
-      <!-- footer>
-        
-          <p>Project maintained by <a href="http://github.com/cxw42">cxw42</a></p>
-        
-        <p><small>Hosted on GitHub Pages &mdash; Theme modified from
-      <a href="https://github.com/pages-themes/leap-day">Leap Day</a> by
-      <a href="https://twitter.com/michigangraham">mattgraham</a></small></p>
-      </footer -->
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+    <title>Tabfern</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="manifest" href="/manifest.json">
+    <link rel="stylesheet" href="/assets/css/styles.min.css">
+<link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet"><link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet">
+</head>
+
+<body style="overflow-x: hidden !important;">
+    <nav class="navbar navbar-light navbar-expand-md navbar-expand-xl sticky-top" style="background-color: white;">
```

Comment:
The responsive nav is very nice!  On mobile, would it be possible to hide the drop-down menu when the user clicks an item in it?  Currently, if I click on "Backup & Restore", I can't see the header because the menu is in the way.  As tested in the Chrome developer tools:

![image](https://user-images.githubusercontent.com/10515894/67628554-b7b07880-f83d-11e9-85e6-7649006d05c9.png)